<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoInput</class>
 <widget class="QDialog" name="InfoInput">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>408</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="firstNameLabel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firstNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lastNameLabel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lastNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="streetAddressLabel">
       <property name="text">
        <string>Street Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="streetAddressLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="cityLabel">
         <property name="text">
          <string>City:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cityLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="stateLabel">
         <property name="text">
          <string>State:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="stateComboBox">
         <item>
          <property name="text">
           <string>--Enter State--</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Alabama</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Alaska</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Arizona</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Arkansas</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>California</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Colorado</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Connecticut</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Delware</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Florida</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Georgia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hawaii</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Idaho</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Illinois</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Indiana</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Iowa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Kansas</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Kentucky</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lousiana</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Maine</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Maryland</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Massachusetts</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Michigan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Minnesota</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mississippi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Missouri</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Montana</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nebraska</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nevada</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Hampshire</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Jersey</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Mexico</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New York</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>North Carolina</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>North Dakota</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ohio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Oklahoma</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Oregon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pennsylvania</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rhode Island</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>South Carolina</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>South Dakota</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tennessee</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Texas</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Utah</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vermont</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Virginia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Washington</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>West Virginia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wisconsin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wyoming</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="zipCodeLabel">
         <property name="text">
          <string>Zip Code:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="zipCodeLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Ignored</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="notesLabel">
         <property name="text">
          <string>Notes:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>188</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="notesTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>278</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
